cmd/gaia/app/extension.go

```diff
@@ -213,6 +213,7 @@ func TxsBlockerForBlock(block tm.Block) func(*GaiaApp, *sql.DB, sdk.Context, typ
 				fmt.Printf("EXECUTING TX %s FOR %d\n", txHash, block.Header.Height)
 				result := app.BaseApp.DeliverTx(tx) // cause transaction to be applied to snapshotted db, so we can interrogate results.
 				jsonTags, _ := app.GetCodec().MarshalJSON(sdk.TagsToStringTags(result.GetTags()))
+				jsonMsgs, _ := app.GetCodec().MarshalJSON(sdktx.GetMsgs())
```

Comment:
This needs some function to convert the transaction messages to JSON like the `TagsToStringTags` in the line above?